ONSdigital/census-rm-acceptance-tests-2021
# Google redis instance

## Motivation and Context
We're currently using a Redis instance deployed as a replica set in our Kubernetes cluster which loses all of its data whenever the pod is killed. Moving to a Memorystore Redis instance will give us a more reliable data cache, and the option for high availability if and when it's required.

## What has Changed
- Get redis config from config map in concourse task

## How to test
Terraform using this branch https://github.com/ONSdigital/census-rm-terraform/pull/33 to create the redis instance and config, deploy the apps using this branch https://github.com/ONSdigital/census-rm-kubernetes/pull/23 of Kubernetes config and run the acceptance tests off this branch. The apps should all function and connect to redis normally

## Links 
https://trello.com/c/zRznW34p/569-use-google-memorystore-redis-instance
https://github.com/ONSdigital/census-rm-terraform/pull/33
https://github.com/ONSdigital/census-rm-kubernetes/pull/23
